src/views/MoonpayHistory.vue

```diff
@@ -129,8 +129,10 @@ export default class MoonpayHistory extends Mixins(mixins.PaginationSearchMixin,
     this.withApi(async () => {
       this.initMoonpayApi(); // MoonpayBridgeInitMixin
 
+      this.setSelectedEvmNetwork(this.ethBridgeEvmNetwork);
```

Comment:
moonpay also should be connected only to ETH network...

Reply:
Yes, this action selects `ethBridgeEvmNetwork` (HASHI bridge network)